content/news/XXXX-XX-XX-mixxx-2-5-beta-released.md

```diff
@@ -0,0 +1,51 @@
+title: Mixxx 2.5 beta released
+authors: Evelynne Veys
+tags: 2.5, beta, release announcement
+comments: yes
+status: draft
+
+#### Dear Mixxx-ers
+
+Funny things happen in spring, birds start nesting and lay eggs, and so did the Mixxx developers.
+A bit late for Easter but it is a magnificent giant gift filled with smaller gifts: Mixxx 2.5 beta!
+You can find the details about the new version below.
+
+#### Appeal
+
+Before releasing 2.5 as the next stable release, we need it to be tested thoroughly, something we can't do on our own.
+After some time looking at the same release and searching for solutions, we might forget minor bugs or problems.  
+That's why we need **YOU** to help us. The more people testing this Beta and reporting eventual bugs to the [bug tracker](https://github.com/mixxxdj/mixxx/issues), the faster the beta can be promoted to stable.  
+In case you're scared of messing up your neatly configured PC and losing your holy data is your Haloween nightmare, read the new wiki article [Safeguard Your Mixxx Data](https://github.com/mixxxdj/mixxx/wiki/Safeguard-your-Digital-DJ-Data).
+It explains about making a timestamp-image, backing up your data and offers tips and tricks.  
+A beta release contains the solutions for problems that occurred and were notified by users who tested the alpha release.
+The beta release is a general check before the new version can be released.  
+The Mixxx database and settings of Mixxx 2.4.x stable are equal to those in Mixxx 2.5 beta, which means you can install the 2.5 over the 2.4 and vice versa without a problem.  
+After you have successfully tested this 2.5 beta for yourself, you can continue using this version in your bedroom or even during live DJ-ing, and thus benefit from the new features.  
```

Comment:
Hmm, not sure what this should tell me. Users should test any version before going live.

Reply:
I think the essence should be, that there is nothing against using a beta live, when you have tested it yourself and you like to benefit from a new feature. Do you have an idea to phrase this nice?

Reply:
indeed, any version should be tested, but using an alpha and certainly a beta doesn't mean that your computer can explode. I wanted to explain that a beta is a very good version, with minor bug that need to be corrected before the stable release.